monai/transforms/intensity/dictionary.py

```diff
@@ -1020,6 +1017,58 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
         return d
 
 
+class RandGibbsNoised(RandomizableTransform, MapTransform):
+    """
+    Dictionary-based wrapper for RandGibbsNoise.
+
+    Naturalistic image augmentation via Gibbs artifacts. The transform 
+    randomly applies Gibbs noise to 3D MRI images. Gibbs artifacts
+    are one of the common type of type artfifacts appearing in MRI scans.
+    
+    For general information on Gibbs artifacts, please refer to:
+    https://pubs.rsna.org/doi/full/10.1148/rg.313105115
+    https://pubs.rsna.org/doi/full/10.1148/radiographics.22.4.g02jl14949
+    
+    Args:
+        prob: probability of applying the transform.
+        keys: 'image', 'label', or ['image', 'label'] depending on which data
+                you need to transform.
+        alpha: Parametrizes the intensity of the Gibbs noise filter applied. Smaller
+            values of alpha correspond to milder applications. If a length-2 list is 
+            given as [a,b] then the value of alpha will be sampled uniformly from the 
+            interval [a,b]. 
+        allow_missing_keys: don't raise exception if key is missing.
+
+    """
+
+    def __init__(
+        self,
+        prob: float = 0.1,
+        keys: KeysCollection = "image",
```

Comment:
```suggestion
        keys: KeysCollection,
        prob: float = 0.1,
```
think all MONAI map transforms are keys first, with no default value.

Reply:
ok, thanks